pyproject.toml

```diff
@@ -11,5 +11,6 @@ exclude = '''
   | debug
   | debian
   | source/third_party
-)/
+  | source/navitiacommon/navitiacommon/.*_pb2\.py
```

Comment:
You also have chaos proto

Reply:
I did fix the version used for black in this PR, some of us didn't have the same version.